meta-leda-components/recipes-sdv/eclipse-leda/sdv-example-containers.bb

```diff
@@ -49,10 +50,10 @@ do_install:append() {
 PACKAGES = "${PN}"
 
 FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DIR}"
-FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DEV_DIR}"
 
 FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DIR}/hvac.json"
 FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DIR}/feedercan.json"
+FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DIR}/feedergps.json"
```

Comment:
We have a distro feature for GPS integration?
The default configuration of that container uses a locally connected gps device, but it can also be configured to access a gpsd provided via network without any locally attached hardware. so that shouldn't be an issue.

Reply:
Then it's okay to install with the other example containers IMO.